recipe/meta.yaml

```diff
@@ -10,23 +10,21 @@ source:
   sha256: 6488eb642acaaf08d8020f6de0a38acee7ac324c1e6e92ee0c0fea42422cb797
 
 build:
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
-    - cross-python_{{ target_platform }}   # [build_platform != target_platform]
-    - python                               # [build_platform != target_platform]
-    - numpy                                # [build_platform != target_platform]
-    - pybind11 >=2.10.0,<2.11.dev0         # [build_platform != target_platform]
   host:
     - python
-    - numpy
-    - pybind11 >=2.10.0,<2.11.dev0
-    - setuptools >=67.6.0,<67.7.dev0
+    - numpy >=1.21.2  # [py<311]
+    - numpy >=1.23.3  # [py>=311]
```

Comment:
in cbc.yaml we have 
```
  # python 3.10
  - 1.21
  # python 3.11
  - 1.23
```
maybe then
numpy {{ numpy }} 
will be better ?

Reply:
That pinning could pull in 1.21.1 or 1.23.1 which would be incompatible. Declaring it explicitly prevents that.

Reply:
Actually, in that case it is better to use`numpy {{ numpy }}`.
>=1.21 will take the latest in main, which is 1.25.
`numpy {{ numpy }}` is the same as `numpy 1.21` or `numpy 1.23`.
For both it will pull the latest 1.21 or 1.23 in main. which is higher than 1.21.2 and 1.23.3